dotcom-rendering/src/lib/decideLogo.ts

```diff
@@ -4,10 +4,6 @@ import type { Branding } from '../types/branding';
 const shouldUseLogoForDarkBackground = (format: ArticleFormat): boolean => {
 	if (format.theme === ArticleSpecial.SpecialReport) return true;
 	switch (format.design) {
-		case ArticleDesign.Gallery:
-		case ArticleDesign.Audio:
-		case ArticleDesign.Video:
-			return true;
```

Comment:
Oh that makes sense. I guess it will take some time to sink in. Thank you Charlotte for taking this work based on your contextual knowledge and as a result of the major UI changes by Fairground.